test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -155,6 +156,7 @@ public void testVerifyColumnNames_sqlException() throws Exception {
 
   @Test
   public void testVerifyColumnNames_syntaxError() throws Exception {
+    assumeFalse("Skip test on SQL Server", is(SQLSERVER));
```

Comment:
Please improve these messages: "SQL Server syntax error does not return SQL state."